IdentityCore/src/broker_operation/request/MSIDBaseBrokerOperationRequest.h

```diff
@@ -34,6 +34,11 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, class, readonly) NSString *operation;
 @property (nonatomic) NSUUID *correlationId;
 
+@property (nonatomic, readonly) NSString *callerBundleIdentifier;
```

Comment:
What about non-native browser request? Will those just be NULL there? I don't see implementation for those in this PR

Reply:
It is nil by default, and once it is nil, we fallback to ASAuthorizationProviderExtensionAuthorizationRequest implementation, see in [broker](https://github.com/AzureAD/azure-activedirectory-tokenbroker-for-objc/blob/e4705525f2d714062accab6e94366e14f16193ec/ADAuthenticationBroker/ADAuthenticationBroker/ASAuthorizationProviderExtensionAuthorizationRequest%2BADBroker.m).

Reply:
Thanks, that's where having more description in the PR would help :)